test/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+if (CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo)
+    set(TEST_BUILD_TYPE --test)
+elseif (CMAKE_BUILD_TYPE STREQUAL Debug)
+    set(TEST_BUILD_TYPE --debug)
+endif ()
+
+if (NO_ICU)
+    set(TEST_EXCLUDE --not-tag exclude_noicu)
+    # TODO any other excludes, for example exclude_icu62AndAboveTestFailures?
```

Comment:
I've had a look at exclude_icu62AndAboveTestFailures, it is used to exclude 2 tests, both of which are already excluded by exclude_noicu. I suppose at some point these broke on newer versions of ICU perhaps?

However these tests now pass when linked to ICU 63.2, so I assume the flag is defunct and should be removed.

Reply:
Thank you, I am going to remove that comment then.

On a related note, aside from embedding ICU, we don't have a way to control its version, but hopefully that can be fixed with ICU detection.